src/NuGet.Clients/NuGet.Console/WpfConsole/WpfConsole.cs

```diff
@@ -600,7 +610,7 @@ public void NavigateHistory(int offset)
             }
         }
 
-        private void WriteProgress(string operation, int percentComplete)
+        private async Task WriteProgressAsync(string operation, int percentComplete)
```

Comment:
Rule #1. https://github.com/microsoft/vs-threading/blob/master/doc/threading_rules.md#rule-1-if-a-method-has-certain-thread-apartment-requirements-sta-or-mta-it-must-either

An async method needs to switch, not check.

Reply:
Ok. Addressed